channel/consensus_channel/consensus_channel.go

```diff
@@ -161,6 +162,26 @@ func (a Add) RightDeposit() big.Int {
 
 var ErrIncorrectTurnNum = fmt.Errorf("incorrect turn number")
 
+// AsState returns a state constructed from the vars and the provided fixed part
+func (vars Vars) AsState(fp state.FixedPart) state.State {
```

Comment:
It feels a bit strange for `AsState` to be a function of `Vars`. It seems more of a `State` constructor.